portfolio/src/main/webapp/index.html

```diff
@@ -23,14 +23,15 @@ <h1>Tal's Portfolio</h1>
       <p>Click here to get a random fact about me!:</p>
       <button id = "button" onclick="addRandomFact()">About Me</button>
       <div id="fact-container"></div>
-      <p>Click here to see a random greeting:</p>
-      <button onclick="getRandomGreeting()">Greet Me!</button>
-      <div id="quote-container"></div>
-
+ 
+      <p>Post a comment here!</p>
       <form method="POST" action="/data">
-        <input type="text" name="title" />
-          <br/>
-      </form>
+      <input type="text" name="text" />
+      <br/>
+      <button>Submit Comment</button>
```

Comment:
This button I think would need a `type="submit"` tag to tell the form that clicking the button submits the form?